suplemon/modules/config.py

```diff
@@ -7,14 +7,26 @@ class Config(Module):
     """Shortcut to openning the current config file."""
 
     def run(self, app, editor, args):
-        path = app.config.path()
-        f = app.file_is_open(path)
-        if f:
-            app.switch_to_file(app.get_file_index(f))
-        else:
-            if not app.open_file(path):
-                app.new_file(path)
+        if args == "defaults":
+            # Open the default config in a new file only for viewing
+            path = app.path + "/config/defaults.json"
```

Comment:
We should prob use `path.join` or concat with `os.path.sep` to avoid complications for Windows